Source/Core/VideoCommon/RenderBase.h

```diff
@@ -390,6 +394,10 @@ class Renderer
   u32 m_last_xfb_stride = 0;
   u32 m_last_xfb_height = 0;
 
+  // Provides a sink for bounding values reads and writes when the feature is disabled or
+  // unsupported, this way games shouldn't crash due to garbage values.
```

Comment:
Might be wise to expand upon this comment for the benefit of future developers.

Something like this?

```suggestion
  // Nintendo's SDK seems to write "default" nonsensical bounding box values before every
  // draw (1023 0 1023 0 are the only values encountered so far), which presumably it is
  // using to detect whether GX has updated the registers with real values.
  //
  // We can store these values when Bounding Box emulation is disabled and return them
  // on read, which the game will interpret as "no pixels have been drawn" 
  // 
  // This produces much better results than just returning garbage, which can cause games
  // like SpiderMan 2 to crash.  
```